webapp/src/services/profilelist/profilelist.service.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+import { authHeader } from "../base.service";
+
+const baseUrl = process.env.REACT_APP_API_URL;
+
+export const profileService = {
+    getProfilesList
+}
+
+function getProfilesList(event_id){
+    return axios
+        .get(baseUrl+"/api/v1/userprofilelist?event_id="+event_id, { headers: authHeader()})
+        .then((response)=>{
+            return { List: response.data, error: ""};
+         })
+        .catch((error)=>{
+            //handling an error, then assign an empty / null array
+            return [], { error:error.message }
```

Comment:
Shouldn't this be something like {List: [], error: error.message} ? 
Also, it's better to set error like this to ensure that you capture any status message from the server first and then fall back to the generic error message:
```
error: error.response && error.response.data ? error.response.data.message : error.message
```